projects/wave-core-new/ng-package.json

```diff
@@ -2,44 +2,7 @@
     "$schema": "../../node_modules/ng-packagr/ng-package.schema.json",
     "dest": "../../dist/wave-core",
     "lib": {
-        "entryFile": "src/public-api.ts",
-        "umdModuleIds": {
-            "codemirror": "codemirror",
-            "d3": "d3",
-            "dagre-d3": "dagreD3",
-            "immutable": "immutable",
-            "moment": "moment",
-            "ngx-color-picker": "ngxColorPicker",
-            "ol/Collection": "OlCollection",
-            "ol/Map": "OlMap",
-            "ol/View": "OlView",
-            "ol/extent": "extent",
-            "ol/format": "format",
-            "ol/format/GeoJSON": "OlFormatGeoJSON",
-            "ol/geom/GeometryType": "OlGeometryType",
-            "ol/geom/Point": "OlGeomPoint",
-            "ol/interaction/Draw": "OlInteractionDraw",
-            "ol/interaction/Select": "OlInteractionSelect",
-            "ol/layer": "layer",
-            "ol/layer/Image": "OlLayerImage",
-            "ol/layer/Tile": "OlLayerTile",
-            "ol/layer/Vector": "OlLayerVector",
-            "ol/proj/Projection": "OlProjection",
-            "ol/proj": "proj",
-            "ol/proj/proj4": "olproj4",
-            "ol/source": "source",
-            "ol/source/ImageStatic": "OlImageStatic",
-            "ol/source/OSM": "OlSourceOSM",
-            "ol/source/TileWMS": "OlTileWmsSource",
-            "ol/source/Vector": "OlSourceVector",
-            "ol/source/XYZ": "XYZ",
-            "ol/style": "style",
-            "ol/style/Fill": "OlStyleFill",
-            "ol/style/Stroke": "OlStyleStroke",
-            "ol/style/Style": "OlStyleStyle",
-            "proj4": "proj4",
-            "vega-embed": "vegaEmbed"
-        }
```

Comment:
Just for curiosity:
Why could you remove all of this and why are there no more UMD warnings about this?

Reply:
the update script did that. I read somewhere that it is no longer required

Reply:
fbd735e